www/www-deploy.yml

```diff
@@ -142,6 +142,26 @@
     - match: "~/site/products/(?<link>.*)$"
       dest: https://www-legacy.openmicroscopy.org/site/products/$link
 
+    # OMERO features
+    - match: "~/omero/new/"
```

Comment:
The `~` i.e. the usage  of regexp,  might be unnecessary here since we are testing exact URLs - unless we want to handle URLs with and without trailing slash i.e. "~/omero/new/?$"